tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -26,7 +26,9 @@ import {
 } from '../../../src/util/logger';
 
 const defaultChromeFlags = ChromeLauncher.defaultFlags()
```

Comment:
I was curious about which are the other default flags in that list and if we should filter out more of them, and so I took a quick look.

The following ones may be worth a look:
 
- `--disable-component-extensions-with-background-pages` seems to have been added because of a puppeteer bug (See https://github.com/puppeteer/puppeteer/issues/4300), but I'm for leave it for now (until we eventually get a report, if it breaks some use case that we are not currently aware of)
- `--disable-sync` may be another flag that it may be reasonable to filter out (e.g. in case an extension developer wants to test something related to the storage.sync API)

@Rob--W do you have any opinion on these two?

Reply:
The `--disable-component-extensions-with-background-pages  flag should be kept. It disables component extensions, i.e. built-in extensions that provide some functionality. Extensions are probably not relying on this.

I don't have a strong opinion on `--disable-sync`. It is not needed for `chrome.storage.sync`, because that API just stores the data locally if there is no synced account. I would be surprised if extensions need sync for testing, as it only makes the test more unpredictable. I'd keep sync disabled for now, until someone requests it to be enabled.

Reply:
@Rob--W great, that is also my current opinion on both. Let's not filter them out yet.